senseicore/src/node.rs

```diff
@@ -1380,6 +1381,23 @@ impl LightningNode {
         Ok((valid, pubkey.to_string()))
     }
 
+    pub fn list_unspent(&self) -> Result<Vec<Utxo>, Error> {
+        let wallet = self.wallet.lock().unwrap();
+        let local_utxos = wallet.list_unspent()?;
+        let utxos = local_utxos
+            .into_iter()
+            .filter(|u| !u.is_spent)
```

Comment:
shouldn't this not be needed because you'd think bdk's `list_unspent` already does this?

Reply:
Ah yes, looks like it does!